frontend/src/components/CoronaActivitiesComponents/ProvinceCapitalWeather.js

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components/macro";
+import {useEffect, useState} from "react";
+import {getWeatherDetails} from "../../service/weatherApiService";
+
+export default function ProvinceCapitalWeather({province}) {
+
+    const [weatherDetails, setWeatherDetails] = useState()
+
+    useEffect(() => {
+        getWeatherDetails(province).then(setWeatherDetails)
```

Comment:
Catch a potential error to prevent your app from crashing

Reply:
not needed because no information will be shown in case of error